docs/schema_guide.md

```diff
@@ -1,157 +1,29 @@
-just be free text. This doesn't follow the schema because it needs to be in `cell_type_ontology_term_id` and
-`cell_type`, and it needs ontology terms and labels, not just any text. In that case the field can be a dictionary:
-
-```
-cell_type_ontology_term_id:
-  CellType:
-    t-cell: CL:0000084
-    b-cell: CL:0000236
-```
-
-This will look at the `obs.CellType` field in the dataset, and where it has the value "t-cell", it will insert
-`CL:0000084` into `cell_type_ontology_term_id` and its label `T cell` into `cell_type`.
-
-Now there are often situations where there is no valid ontology term for some field. For example, the dataset may have
-been produced via an assay not present in `EFO`. Or, a particular cell type may have no entry in `CL`. In that case, a
-free text description can be used in the `ontology_term_id` field:
-
-```
-assay_ontology_term_id: Sci-Plex
-cell_type_ontology_term_id:
-  CellType:
-    t-cell: CL:0000084
-    b-cell: CL:0000236
-    new cell type: new cell type
-```
-
-In these cases, the `cellxgene-schema apply` script will leave the ontology field blank and move the free text
-description into the label field. So the `assay_ontology_term_id` in the new dataset would be `""` but `assay` would be
-`Sci-Plex`.
-
-
-#### Gene symbol harmonization
-
-The last section describes how gene symbol conversion should be applied to each of the layers. This is similar to the
-`layer_descriptions` field above, but there are only three permitted values: `raw`, `log1p`, and `sqrt`:
-
-```
-fixup_gene_symbols:
-  X: log1p
-  raw.X: raw
-```
-
-This tells the script how each each layer was transformed from raw values that can be directly summed. `raw` means that
-the layer contains raw counts or some linear tranformation of raw counts. `log1p` means that the layer has `log(X + 1)`
-for each the raw `X` values. `sqrt` means `sqrt(X)` (this is not common). For layers produced by Seurat's normalization
-or SCTransform functions, the correct choice is usually `log1p`.
-
-
-### `cellxgene-schema validate`
-
-The next `cellxgene-schema` subcommand is `cellxgene-schema validate`, and it validates that a given h5ad follows a
-version of the schema. It accepts two parameters:
-
-1. The h5ad file to check
-2. Whether the validation should be "shallow"
 
-If the validation is shallow, all the script does is check for a valid schema version number in the `uns` metadata. This
-works because the `cellxgene-schema apply` command won't write the version into the file unless it verifies that the new
-h5ad file does indeed follow the schema. If the validation is "deep", then it reads the schema version and actually checks
-all the fields in the h5ad against it.
+This execution will validate the dataset as above AND will add the human-readable labels associated to ontology and gene IDs as defined in the schema. If the validation is successful a new AnnData file (`output.h5ad`) will be written to disk with the labels appended. 
```

Comment:
This execution validates the dataset as above AND adds the human-readable labels for the ontology and gene IDs as defined in the schema. If the validation is successful, a new AnnData file (`output.h5ad`) is written to disk with the labels appended.